src/hotspot/share/runtime/sharedRuntime.cpp

```diff
@@ -3014,11 +3014,6 @@ void AdapterHandlerLibrary::create_native_wrapper(const methodHandle& method) {
   assert(method->is_method_handle_intrinsic() ||
          method->has_native_function(), "must have something valid to call!");
 
-  if (CriticalJNINatives && !method->is_method_handle_intrinsic()) {
-    // We perform the I/O with transition to native before acquiring AdapterHandlerLibrary_lock.
-    critical_entry = NativeLookup::lookup_critical_entry(method);
```

Comment:
`critical_entry` variable is now redundant?

Reply:
removed, thanks for spotting that.